STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/ErrorMappers/ErrorMapperServiceTests.Logic.Retrieve.cs

```diff
@@ -0,0 +1,45 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using STX.REST.RESTFulSense.Clients.Models.Errors;
+using Tynamix.ObjectFiller;
+using Xunit;
+
+namespace STX.REST.RESTFulSense.Clients.Tests.Unit.Services.Foundations.ErrorMappers
+{
+    public partial class ErrorMapperServiceTests
+    {
+        [Fact]
+        private async Task ShouldRetrieveStatusDetailByStatusCodeAsync()
+        {
+            //given
```

Comment:
```suggestion
            // given
```
Do the same for the rest.

Reply:
Fixed

Reply:
@aosunlana this also includes when & then